docs/apis/streaming/connectors/kinesis.md

```diff
@@ -26,7 +26,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 
-The Kinesis connector provides access to [Amazon AWS Kinesis Streams](http://aws.amazon.com/kinesis/streams/). 
```

Comment:
My editor did this automatically and I didn't catch it. Again, no more Friday afternoon pushes for me.